variables.tf

```diff
@@ -216,6 +222,58 @@ variable "kubernetes_version" {
   default     = null
 }
 
+variable "load_balancer_profile_idle_timeout_in_minutes" {
+  type        = number
+  description = "(Optional) Desired outbound flow idle timeout in minutes for the cluster load balancer. Must be between `4` and `120` inclusive."
+  default     = null
```

Comment:
According to the [document](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/kubernetes_cluster#idle_timeout_in_minutes), could we set the default value to `30` here?